includes/wf_crm_webform_preprocess.inc

```diff
@@ -183,12 +183,21 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
         'scope' => 'footer',
       );
       $page_id = $this->data['contribution'][1]['contribution'][1]['contribution_page_id'];
-      $contributionCallbackQuery = array('reset' => 1, 'id' => $page_id, 'qfKey' => $this->getQfKey(), 'snippet' => 4);
+      if ($this->civicrm_version >= '4.7') {
+        $currency = '$'; // FIXME
```

Comment:
This value could be retrieved from the contribution page. In fact I think we already have the contribution page loaded in memory at this point.